### PR TITLE
fix stack size stuff

### LINKED ISSUE #16: [BUG] Illegal stack size: 3
## Description (REQUIRED)

The LiteXpansion plugin writes warning to console when loading.

## Steps to reproduce the Issue (REQUIRED)

Start server with LiteXpansion.

## Expected behavior (REQUIRED)

Fix this warning.

## Server Log

https://pastebin.com/dijHMfMC

## Environment (REQUIRED)

[20:33:23 INFO]: This Server uses the following setup of Slimefun:
[20:33:23 INFO]: Yatopia git-Yatopia-"86c42a4" (MC: 1.16.4)
[20:33:23 INFO]: CS-CoreLib vDEV - 92 (git e9150d72)
[20:33:23 INFO]: Slimefun vDEV - 784 (git ac590fdd)
[20:33:23 INFO]: Metrics build: #19
[20:33:23 INFO]: 
[20:33:23 INFO]: Installed Addons: (20)
[20:33:23 INFO]:  DyedBackpacks vDEV - 13 (git 964b2d55)
[20:33:23 INFO]:  ExtraTools vDEV - 23 (git 9299a65c)
[20:33:23 INFO]:  PomaExpansion vDEV - 13 (git 78eb8cc5)
[20:33:23 INFO]:  FoxyMachines v1.0.0
[20:33:23 INFO]:  SlimefunWarfare vDEV - 11 (git 001a2946)
[20:33:23 INFO]:  EcoPower vDEV - 3 (git 18ba903d)
[20:33:23 INFO]:  HotbarPets vDEV - 34 (git ee2130bb)
[20:33:23 INFO]:  ExtraGear vDEV - 12 (git 4570ea2c)
[20:33:23 INFO]:  SlimefunLuckyBlocks vDEV - 30 (git 26824119)
[20:33:23 INFO]:  Liquid vDEV - 6 (git 319b40ca)
[20:33:23 INFO]:  InfinityExpansion vDEV - 58 (git 099b2a4d)
[20:33:23 INFO]:  RestrictedCreative v2.4.5
[20:33:23 INFO]:  SlimeCustomizer vDEV - 1 (git 30217f4a)
[20:33:23 INFO]:  SFCalc vDEV - 2 (git da4a9d46)
[20:33:23 INFO]:  LiteXpansion vDEV - 119 (git 919c82ec)
[20:33:23 INFO]:  SlimyTreeTaps vDEV - 17 (git 8f3e755b)
[20:33:23 INFO]:  MobCapturer vDEV - 13 (git 79bbd87e)
[20:33:23 INFO]:  TranscEndence vDEV - 22 (git 7660c9f0)
[20:33:23 INFO]:  FluffyMachines vDEV - 59 (git fa52f3b1)
[20:33:23 INFO]:  SlimyRepair vDEV - 2 (git 1a0065a5)

### PR DESCRIPTION
**Pull request by Riley31415**
## Short Description
fixed the stack size error/complaints

## Additions/Changes/Removals
added a new method to register unpeaceable block with output amount greater than 1

## Related Issues
Resolves #16

## Checklist

- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I have also tested the proposed changes in combination with base Slimefun and made sure nothing breaks/unexpected happens.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added which may not be obvious to maintainers.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java [hunk 1]
```diff
@@ -322,7 +322,24 @@ private void registerNonPlaceableItem(@Nonnull SlimefunItemStack result, @Nonnul
             recipe = items;
         }
 
-        new UnplaceableBlock(Items.LITEXPANSION, result, type, recipe).register(plugin);
+        new UnplaceableBlock(Items.LITEXPANSION, item, type, recipe).register(plugin);
+    }
+
+    /**
+     * Register an item with an output amount greater than 1
```
Comment (J3fftw1) [inline src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:320]: 1 line comment, please use //

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

**Comment by WalshyDev** [comment]
Make the existing method use the new one. You're breaking DRY.

File: src/main/java/dev/j3fftw/litexpansion/ItemSetup.java (hunk 1 above)
Comment (WalshyDev) [reply src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:320]: It's a JavaDoc over the method :p

Comment (J3fftw1) [reply src/main/java/dev/j3fftw/litexpansion/ItemSetup.java:320]: We dont generate 1 so who cares

**Pull request merged**